module/batch-action/src/Domain/Model/BatchActionInformationModel.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\BatchAction\Domain\Model;
+
+use Ergonode\BatchAction\Domain\Entity\BatchActionId;
+use Ergonode\BatchAction\Domain\ValueObject\BatchActionType;
+
+class BatchActionInformationModel
+{
+    private BatchActionId $id;
+
+    private BatchActionType $type;
```

Comment:
Shouldn't this by actionType? Since it's a model and it's available via API I guess it's not really clear what the type refers to.

Reply:
Type of batch action, each batch action have own type for identify matching process, use for communication between frontend and backed. We can think about adding endpoint with list of available action types